packages/web-components/src/image/image.styles.ts

```diff
@@ -22,44 +23,44 @@ export const styles = css`
     min-width: 8px;
     display: inline-block;
   }
-  :host([block]) ::slotted(img) {
+  :host(:is([state--block], :state(block))) ::slotted(img) {
```

Comment:
All of the states with exception of shape here seem very specific to image as of now - if any of these end up getting repeated we should pull them out.